Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
```

Comment:
also, please add the rust github actions -> go to "Actions" and search for Rust, use this file
https://github.com/hubble-markets/hubble/blob/master/.github/workflows/rust.yml

but remove the "continue-on-error" to be false

Reply:
done!